buildlib/archtable

```diff
@@ -13,10 +13,10 @@ sparc64	sparc
 alpha.*	alpha
 m68k	m68k
 arm.*	arm
-ppc	powerpc
-powerpc	powerpc
-ppc64		powerpc64
-powerpc64	powerpc64
+ppc	ppc
+powerpc	ppc
+ppc64		ppc64
+powerpc64	ppc64
```

Comment:
I know, @julian-klode told me. However, I still want this stuff merged in so that we have the historical data.